taxonomies/categories.txt

```diff
@@ -1417,41 +1417,304 @@ agribalyse_proxy_food_name:en:Beer, dark
 agribalyse_proxy_food_name:fr:Bière brune
 
 <en:Beers
-en:India Pale Ale (IPA), IPA ales, India pale ale, India pale ales, IPA beers
-es:Cerveza IPA, IPA
+en:Packaged beers, Beers in a package, Packed beers
+lt:Supakuotas alus, Alus pakuotėje, Alus dėžėje
+
+################################# ALES #################################
+
+<en:Beers
+en:Ales, Ale
+lt:Elis
+wikidata:en:Q208385
+
+<en:Ales
+en:Pale ales, Pale ale
+
+############ IPA ############
+
+<en:Pale ales
 ca:Cervesa IPA, IPA
+en:India Pale Ale (IPA), IPA ales, India pale ale, India pale ales
+es:Cerveza IPA, IPA
+fr:Bières IPA, Bieres IPA, Bière ipa
 xx:India Pale Ale (IPA)
```

Comment:
I would suggest to put some synonyms in the xx: definition so that we don't have to repeat them in all languages

```suggestion
xx:India Pale Ale (IPA), India Pale Ale, IPA
```